src/cmdstan-guide/pathfinder_config.Rmd

```diff
@@ -0,0 +1,169 @@
+    "x_center": [0.126047, -0.065048, 1.55708, 0.958509, 0.628075, -0.217041, 1.32032, -0.561338, 1.42988, 1.23213],
+    "logDetCholHk": -2.6839,
+    "L_approx": [[-0.0630456, -0.0187959], [0, 1.08328]],
+    "Qk": [[-0.361073, 0.5624], [0.183922, -0.279474], [-0.0708175, 0.15715], [-0.00917823, 0.0215802], [0.0606019, -0.0814513], [0.164071, -0.285769], [-0.057723, 0.112428], [0.276376, -0.424348], [-0.0620524, 0.131786], [-0.846488, -0.531094]],
+    "alpha": [1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
+    "full": false,
+    "lbfgs_note": ""
+  },
+  ...,
+  "171": {
+    "iter": 171,
+    "unconstrained_parameters": [1.60479, 1.60479, 1.60479, 1.60479, 1.60479, 1.60479, 1.60479, 1.60479, 1.60479, -35.7821],
+    "grads": [2.66927e+15, -0.117312, -0.0639521, -2.66927e+15, -0.0445885, 0.0321579, 0.00499827, -0.163952, -0.032084, 6.4073],
+    "history_size": 5,
+    "lbfgs_success": true,
+    "pathfinder_success": true,
+    "x_center": [5.58876e+15, 5.58876e+15, 5.58876e+15, 5.58876e+15, 5.58876e+15, 5.58876e+15, 5.58876e+15, 5.58876e+15, 5.58876e+15, -2.02979e+17],
+    "logDetCholHk": 299.023,
+    "L_approx": [[4.6852e+06, 4.6852e+06, 4.6852e+06, 4.6852e+06, 4.6852e+06, 4.6852e+06, 4.6852e+06, 4.6852e+06, 4.6852e+06, -1.70162e+08], [0, 2.19511e+13, 2.19511e+13, 2.19511e+13, 2.19511e+13, 2.19511e+13, 2.19511e+13, 2.19511e+13, 2.19511e+13, -7.97244e+14], [0, 0, 2.19511e+13, 2.19511e+13, 2.19511e+13, 2.19511e+13, 2.19511e+13, 2.19511e+13, 2.19511e+13, -7.97244e+14], [0, 0, 0, 2.19511e+13, 2.19511e+13, 2.19511e+13, 2.19511e+13, 2.19511e+13, 2.19511e+13, -7.97244e+14], [0, 0, 0, 0, 2.19511e+13, 2.19511e+13, 2.19511e+13, 2.19511e+13, 2.19511e+13, -7.97244e+14], [0, 0, 0, 0, 0, 2.19511e+13, 2.19511e+13, 2.19511e+13, 2.19511e+13, -7.97244e+14], [0, 0, 0, 0, 0, 0, 2.19511e+13, 2.19511e+13, 2.19511e+13, -7.97244e+14], [0, 0, 0, 0, 0, 0, 0, 2.19511e+13, 2.19511e+13, -7.97244e+14], [0, 0, 0, 0, 0, 0, 0, 0, 2.19511e+13, -7.97244e+14], [0, 0, 0, 0, 0, 0, 0, 0, 0, 2.89552e+16]],
+    "Qk": [],
+    "alpha": [1.11027e-12, 2.24669e-12, 2.05603e-12, 3.71177e-12, 5.7855e-12, 1.80169e-12, 3.40291e-12, 2.29699e-12, 3.43423e-12, 1.25815e-08],
+    "full": true,
+    "lbfgs_note": ""
+  },
+  "172": {
+    "iter": 172,
+    "unconstrained_parameters": [1.60531, 1.60531, 1.60531, 1.60531, 1.60531, 1.60531, 1.60531, 1.60531, 1.60531, -35.801],
+    "grads": [-0, -0.11731, -0.0639469, 0.0179895, -0.0445842, 0.0321643, 0.00500256, -0.163947, -0.0320824, 7],
+    "history_size": 5,
+    "lbfgs_success": false,
+    "pathfinder_success": false,
+    "lbfgs_note": ""
+  }
+}
+```
+
+Option `num_paths=1` runs just one single Pathfinder, in which case the
+output CSV file contains just the sample from that run.
+The combination of arguments `num_paths=1 save_single_paths=1` creates just two output files,
+the CSV sample and the set of ELBO iterations.
+In this case, the default output file name is "output.csv" and the default diagnostic file name is "output.json".
+
```

Comment:
If we run just a single run of single-path pathfinder will it still importance resample?   I think it's OK either way, but should be documented.  Coming at this from the paper, it's not entirely clear how to configure single-path pathfinder, not that I'd recommend anyone doing that.

Reply:
no, single path pathfinder doesn't importance resample.  there are two service methods; for `num_paths=1`,  service `pathfinder_lbfgs_single` is run.

Reply:
reworded to clarify